src/js/components/inspectors/MoreProperties.tsx

```diff
@@ -37,25 +37,25 @@ class BaseMoreProperties extends React.Component<MorePropsProps, MorePropsState>
       moreLink = (
         <div>
           <h3 className='show-more-props more-props-label'
-            onClick={this.handleClick} style={style1}>
+            onClick={(e) => this.handleClick(e)} style={style1}>
```

Comment:
Just a comment because I edited this file too, instead of changing all of the 'this.handleClick', I found you can just add 'this.handleClick = this.handleClick.bind(this);' to the constructor and these will work as is

Reply:
Yeah. So I chose this approach because it was more consistent with InteractionInputType.